### ISSUE TITLE
HTML body is not displayed

### ISSUE THREAD

**Issue by carlptr** [Jaben/Papercut#5]
I use PaperCut to display email notifications from a SharePoint dev environment.

The emails are received as expected, but the HTML body is not displayed.
![papercut](https://cloud.githubusercontent.com/assets/1091126/10509830/bcecbed6-732f-11e5-98a4-b17c28231bac.png)
![papercut2](https://cloud.githubusercontent.com/assets/1091126/10509831/bcf1cd68-732f-11e5-9ce0-eeaf454004c7.png)

When I open the eml with Outlook everything looks good.
I started the application with ClickOnce, therefore I should use the latest version.

Thanks for your support!

**Comment by Jaben** [comment]
Please attach the offending EML file to help reproduce the issue.

**Comment by carlptr** [comment]
Here we go :-)
https://www.dropbox.com/s/9y0uws3idmium5a/20151015-112116-8933ec5.eml?dl=0

**Comment by Jaben** [comment]
@jstedfast No expectation, but the HtmlPreviewVisitor is mangling the html on this email Carlptr attached. I'm investigating. But, if you have a second any help would be appreciated.

**Comment by Jaben** [comment]
@jstedfast Further info, his HTML looks like this (very simple): 
```HTML
<HTML dir=ltr><HEAD><TITLE></TITLE></HEAD><BODY marginheight="0" marginwidth="0" topmargin="0" leftmargin="0"><FONT face=verdana size=2><p>Alert 'Export packages' has successfully been added on <a href="http://coidmo01">'COI-PowerDoc@S-Bahn Berlin'</a>. </p><p>You will receive alerts according to the delivery method, timing and criteria that were selected when the alert was created.</p><p>You can change this alert or any of your other alerts on the <a href="http://coidmo01/_layouts/MySubs.aspx">My Alerts on this Site</a> page.</p></FONT></BODY></HTML>
```
After processing, it becomes:
```html
<!-- saved from url=(0014)about:internet -->
<HTML dir="ltr"><HEAD><TITLE>&lt;/TITLE&gt;&lt;/HEAD&gt;&lt;BODY marginheight=&quot;0&quot; marginwidth=&quot;0&quot; topmargin=&quot;0&quot; leftmargin=&quot;0&quot;&gt;&lt;FONT face=verdana size=2&gt;&lt;p&gt;Alert &#39;Export packages&#39; has successfully been added on &lt;a href=&quot;http://coidmo01&quot;&gt;&#39;COI-PowerDoc@S-Bahn Berlin&#39;&lt;/a&gt;. &lt;/p&gt;&lt;p&gt;You will receive alerts according to the delivery method, timing and criteria that were selected when the alert was created.&lt;/p&gt;&lt;p&gt;You can change this alert or any of your other alerts on the &lt;a href=&quot;http://coidmo01/_layouts/MySubs.aspx&quot;&gt;My Alerts on this Site&lt;/a&gt; page.&lt;/p&gt;&lt;/FONT&gt;&lt;/BODY&gt;&lt;/HTML&gt;
```

**Comment by Jaben** [comment]
@jstedfast Pretty sure the empty <TITLE></TITLE> tag is breaking it. As soon as it goes to title, WriteTag stops processing.

**Comment by jstedfast** [comment]
Ouch, that was a face-palm worthy bug :(

**Comment by Jaben** [comment]
@jstedfast hah, at least it's fixed. :+1:

**Comment by jstedfast** [comment]
I'll make a new MimeKit release after the holidays

**Comment by Jaben** [comment]
@jstedfast awesome -- thanks for looking at it & the fix! Happy Holidays!

**Comment by jstedfast** [comment]
FWIW, I released MimeKit 1.2.19 to nuget.org a few days ago with the fix for this.

**Comment by Jaben** [comment]
Updated. Fixed in latest. Will release here this week.
